src/RichText.xml

```diff
@@ -106,5 +106,11 @@ WARNING disabling this option is a security risk for XSS attacks</description>
             <description>Microflow will be triggered when focus is lost and the content has changed</description>
             <returnType type="Void"/>
         </property>
+        <property key="onChangeNanoflow" type="nanoflow" required="false">
+            <caption>On change nanoflow</caption>
+            <category>Events</category>
+            <description>Nanoflow will be triggered when focus is lost and the content has changed</description>
```

Comment:
i think description should be 
`Nanoflow that will be triggered when focus is lost and the content has changed` and this applies to the microflow